.vscode/settings.json

```diff
@@ -1,3 +1,5 @@
 {
-    "editor.tabSize": 2
+    "editor.tabSize": 2,
+    "editor.wordWrap": "wordWrapColumn",
+    "editor.wordWrapColumn": 100
```

Comment:
I am not a fan of these settings - feel free to enable in your local config, but let's not force on all devs.

Reply:
Okay will do!